### ISSUE TITLE
Image Links fail if the line is folded in the middle of the text

### ISSUE THREAD

**Issue by mementum** [asciidoctor/kramdown-asciidoc#116]
Considering your input from #115, I don't know if this is down to the converter or it shall go upstream.

This will work after being translated:

```markdown
![image text](hello.png)
```

But this will not work

```markdown
![image
text](hello.png)
```

The translation in the 2nd case is:

```asciidoc
image::images/hello.png[image
text]
```

and this later fails when the `asciidoc` source is converted to `pdf` or `epub`. No image will be shown, just the macro and what follows.

The same does not happen for web links, which do work when folded in the same fashion.

**Comment by mojavelinux** [comment]
Indeed, Kramdown AsciiDoc should be removing (normalizing) this newline as a block attribute has to be on a single line.
